Mathlib/Analysis/SpecialFunctions/Complex/CircleAddChar.lean

```diff
@@ -63,6 +63,13 @@ lemma toCircle_apply (j : ZMod N) :
     toCircle j = exp (2 * π * I * j.val / N) := by
   rw [← toCircle_natCast, natCast_zmod_val]
 
+lemma toCircle_eq_circleExp (j : ZMod N) :
```

Comment:
This is already part of your other PR I just reviewed, isn't it? When you have one PR that depends on another you should add a "depends on..." flag to the later PR's in the sequence, so your code isn't getting reviewed twice.

Reply:
> This is already part of your other PR I just reviewed, isn't it? When you have one PR that depends on another you should add a "depends on..." flag to the later PR's in the sequence, so your code isn't getting reviewed twice.

PS. I see you already did. But then it's probably better to wait with requesting reviews until the prerequisite PRs are merged (yes, I know this slows things down, sorry).

Reply:
Sorry.